shark/examples/shark_training/bert_training_load_tf.py

```diff
@@ -0,0 +1,51 @@
+import sys
+from absl import app
+import time
+
+import numpy as np
+import os
+import tempfile
+import tensorflow as tf
+
+from official.nlp.modeling import layers
+from official.nlp.modeling import networks
+from official.nlp.modeling.models import bert_classifier
+
+from shark.shark_trainer import SharkTrainer
+from shark.parser import parser
+
+parser.add_argument(
+    "--target_mlir",
```

Comment:
nit. load_mlir may be better to match save_mlir but that can be changed in a future commit.

Reply:
haha, true, it was initially assuming you have a local mlir file. I can change it to download_mlir_path, so it won't conincide with the save_mlir flag which you need to initially save the mlir.

Reply:
or just mlir_path (since you may not always download it : )